src/apiClient.js

```diff
@@ -743,22 +753,66 @@ class ApiClient {
     }
 
     getDownloadSpeed(byteSize) {
-        const url = this.getUrl('Playback/BitrateTest', {
-            Size: byteSize
-        });
+        return new Promise((resolve, reject) => {
+            const url = this.getUrl('Playback/BitrateTest', {
+                Size: byteSize
+            });
 
-        const now = new Date().getTime();
+            console.log(`Requesting ${url}`);
 
-        return this.ajax({
-            type: 'GET',
-            url,
-            timeout: 5000
-        }).then(() => {
-            const responseTimeSeconds = (new Date().getTime() - now) / 1000;
-            const bytesPerSecond = byteSize / responseTimeSeconds;
-            const bitrate = Math.round(bytesPerSecond * 8);
+            const xhr = new XMLHttpRequest;
+
+            xhr.open('GET', url, true);
+
+            xhr.responseType = 'blob';
+            xhr.timeout = BITRATETEST_TIMEOUT;
+
+            const headers = {
+                'Cache-Control': 'no-cache'
```

Comment:
Should we use `no-store`?

From [mdn](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control#directives):

> Note that no-cache does not mean "don't cache". no-cache allows caches to store a response but requires them to revalidate it before reuse. If the sense of "don't cache" that you want is actually "don't store", then no-store is the directive to use.

Reply:
The old version (AJAX) didn't seem to use any cache control directives. :man_shrugging:
I added both.